src/VisualStudio/Core/Test.Next/project.json

```diff
@@ -2,18 +2,8 @@
   "dependencies": {
     "Newtonsoft.Json": "8.0.3",
     "Moq": "4.2.1402.2112",
```

Comment:
@jasonmalinowski it is what jason suggested as far as I understood. 
adding ForceBindingRedirect and adding d15 dlls explicitly so that it wins over. (I could misunderstood :)) 
...

issues are 2. 

one is we sharing 1 directory for both d14 projects and d15 projects which will dump all its dependencies to same directory. which I believe @mattwar  is working on to fix.

second one is when a project has p2p reference to d14 projects and d15 projects how to make sure references for d15 win. and what I did was the way to win it.